core/domain/classifier_services.py

```diff
@@ -118,3 +122,68 @@ def classify_string_classifier_rule(state, normalized_answer):
             return None
 
     return None
+
+
+def get_classifier_from_model(classifier_model):
+    """Returns a classifier domain object given a classifier model loaded
+    from datastore.
+    """
+    return classifier_domain.Classifier(
+        classifier_model.id, classifier_model.exp_id,
+        classifier_model.exp_version_when_created,
+        classifier_model.state_name, classifier_model.algorithm_id,
+        classifier_model.cached_classifier_data,
+        classifier_model.data_schema_version)
+
+def get_classifier_by_id(classifier_id):
+    """Returns a classifier domain object given a classifier id.
+    """
+    classifier_model = classifier_models.ClassifierModel.get(
+        classifier_id, strict=False)
+    if classifier_model is None:
+        raise Exception("No classifier found for the classifer's id.")
```

Comment:
The error message should state the id.

"Could not find the classifier instance for this id: %s."

(Also, note that the second 'classifier' is currently misspelled.)

Also also, if we are going to raise an exception like this anyway, what is the point of using strict=False?

Similar comments apply elsewhere.